### PR TITLE
Frontend changes for Delivery Statuses

### PR DESCRIPTION
**Pull request by s0l0ist**
## Background

This PR contains the backend AppSync configuration for the frontend to consume. It will be the main PR for the frontend related work #1392

Also fixes issue #1513 

## Changes

- Added resolver for deliverAlert (re-sending alerts)
- Added graphql attributes/fields/types to support frontend invocation

## Testing

- mage test:ci

### CODE REVIEW

File: api/graphql/schema.graphql [hunk 1]
```diff
@@ -25,6 +25,7 @@ type Mutation {
   deleteUser(id: ID!): Boolean
   inviteUser(input: InviteUserInput): User!
   remediateResource(input: RemediateResourceInput!): Boolean
+  deliverAlert(input: DeliverAlertInput!): AlertSummary
```
Comment (3nvi) [inline api/graphql/schema.graphql:28]: @s0l0ist can this response ever be `null`? Why so?

File: api/graphql/schema.graphql [hunk 2]
```diff
@@ -182,16 +186,38 @@ type AlertSummary implements Alert {
...
 input UpdateAlertStatusInput {
   alertId: ID!
   status: AlertStatusesEnum!
 }
 
+input SendTestAlertInput {
+  outputIds: [ID!]!
+}
+
+type SendTestAlertResponse {
```
Comment (3nvi) [inline api/graphql/schema.graphql:216]: @s0l0ist same here, would `success` or `message` not be defined?

File: api/graphql/schema.graphql (hunk 1 above)
Comment (s0l0ist) [reply api/graphql/schema.graphql:28]: We always return an empty array. Slices should never be null, only empty.

File: api/graphql/schema.graphql (hunk 2 above)
Comment (s0l0ist) [reply api/graphql/schema.graphql]: The shape has changed here and has been resolved.

File: api/graphql/schema.graphql (hunk 1 above)
Comment (s0l0ist) [reply api/graphql/schema.graphql]: Fixed

**Review by 3nvi — APPROVED** [review]

**Comment by 3nvi** [comment]
@s0l0ist assuming you verified the functionality, so I'll be merging this

**Pull request merged**